solana.nix

```diff
@@ -0,0 +1,89 @@
+{
+  system ? builtins.currentSystem,
+  pkgs ? import <nixpkgs> { inherit system; },
+}:
+
+# Solana integration
+let
+  version = "v1.18.22";
```

Comment:
would it be good to have automatically upgrade this?

just thinking because there was a new release yesterday to v1.18.23 so this is already a bit outdated 🤷🏼

Reply:
That would be great, I’ve added the modification script for the two files to the script!

- Version update script for each features to update-solana.sh [c57bed1 (#839)](https://github.com/smartcontractkit/chainlink-solana/pull/839/commits/c57bed1e4dea10ab1764a3f293f05991e4fa0e37)
- Rename upgrade-e2e-solana-image >> upgrade-solana-image for better context [e4744c5 (#839)](https://github.com/smartcontractkit/chainlink-solana/pull/839/commits/e4744c57432067f4e556e008bba9a71da9d08712)

Reply:
- Update nix hashes on version update [`7ff50e2` (#839)](https://github.com/smartcontractkit/chainlink-solana/pull/839/commits/7ff50e2ab5933ef961c002554f3b38bb7260e4b2)